odinmain/src/main/kotlin/me/odinmain/features/impl/floor7/WitherDragonEnum.kt

```diff
@@ -40,6 +54,46 @@ enum class WitherDragonsEnum (
 
     None(Vec3(0.0, 0.0, 0.0), AxisAlignedBB(0.0, 0.0, 0.0, 0.0, 0.0, 0.0), 'f', Color.WHITE,
         0.0..0.0, 0.0..0.0);
+
+    fun setAlive(entityId: Int) {
+        state = WitherDragonState.ALIVE
+
+        timeToSpawn = 100
+        timesSpawned += 1
+        this.entityId = entityId
+        spawnedTime = System.currentTimeMillis()
+        isSprayed = false
+
+        if (sendArrowHit && WitherDragons.enabled) arrowSpawn(this)
+        if (resetOnDragons && WitherDragons.enabled) onDragonSpawn()
+        if (sendSpawned && WitherDragons.enabled) {
+            val numberSuffix = when (timesSpawned) {
```

Comment:
There is probably a kotlin std function to do this.

Reply:
I couldn't find one